tools/shared/src/main/scala/org/scalajs/core/tools/optimizer/Linker.scala

```diff
@@ -38,50 +38,51 @@ final class Linker(semantics: Semantics, outputMode: OutputMode,
   def this(semantics: Semantics, considerPositions: Boolean) =
     this(semantics, OutputMode.ECMAScript51Isolated, considerPositions)
 
-  private[this] val files = mutable.Map.empty[String, PersistentIRFile]
-
-  private[this] var statsReused: Int = 0
-  private[this] var statsInvalidated: Int = 0
-  private[this] var statsTreesRead: Int = 0
-
   type TreeProvider = String => (ClassDef, Option[String])
 
-  /** Cleans the cache. */
-  def clean(): Unit = {
-    files.clear()
-    statsReused = 0
-    statsInvalidated = 0
-    statsTreesRead = 0
+  /** Obsolete method. Does not do anything. */
+  @deprecated("Does not do anything. Don't use", "0.6.5")
+  def clean(): Unit = ()
+
+  // Binary compat
+  def link(irInput: immutable.Traversable[VirtualScalaJSIRFile], logger: Logger,
```

Comment:
Please add "TODO" in the comment, so that this can be found when we grep for "TODO"s and "deprecated"s one day.